app/uk/gov/hmrc/cgtpropertydisposalsfrontend/controllers/returns/yeartodatelliability/YearToDateLiabilityController.scala

```diff
@@ -1050,7 +1070,7 @@ class YearToDateLiabilityController @Inject() (
     draftReturn: DraftReturn
   )(implicit request: RequestWithSessionData[_]): Future[Result] =
     answers match {
-      case IncompleteNonCalculatedYTDAnswers(None, _, _) =>
+      case IncompleteNonCalculatedYTDAnswers(None, _, _) if !isATrust(fillingOutReturn) =>
```

Comment:
we'd have to double check but I'm not sure if we should skip this question for trusts

Reply:
just confirmed, we don't skip this page for trusts

Reply:
Done.